packages/vite/src/node/plugins/worker.ts

```diff
@@ -45,31 +46,32 @@ export function webWorkerPlugin(config: ResolvedConfig): Plugin {
         url = await fileToUrl(cleanUrl(id), config, this)
         url = injectQuery(url, WorkerFileId)
       } else {
-        if (query.inline != null) {
-          // bundle the file as entry to support imports and inline as base64
-          // data url
-          const rollup = require('rollup') as typeof Rollup
-          const bundle = await rollup.rollup({
-            input: cleanUrl(id),
-            plugins: config.plugins as Plugin[]
+        // bundle the file as entry to support imports
+        const rollup = require('rollup') as typeof Rollup
+        const bundle = await rollup.rollup({
+          input: cleanUrl(id),
+          plugins: config.plugins as Plugin[]
+        })
+        try {
+          const { output } = await bundle.generate({
+            format: 'iife',
```

Comment:
**question:** Seeing that this changed from 'es' to 'iife', what does this mean? Is it a breaking change? Is it backwards compatible?

Reply:
I changed this from es to iife because Firefox was running into trouble with default exports still being exported (something like `unexpected top level export`), so this was how I got rid of those imports and exports. I believe this module is standalone because you can't import code from a worker bundle so afaik this is not a breaking change.